x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -469,15 +482,24 @@ export class ManifestManager {
       if (isEmpty(artifactIds)) {
         return [];
       }
+
       const errors = await this.artifactClient.bulkDeleteArtifacts(artifactIds);
+
       if (!isEmpty(errors)) {
         return errors;
       }
-      for (const artifactId of artifactIds) {
-        this.logger.info(`Cleaned up artifact ${artifactId}`);
+
+      this.logger.info(`Count of cleaned up artifacts: ${artifactIds.length}`);
+
+      if (artifactIds.length !== 0) {
+        this.logger.debug(`Deleted artifacts from cleanup:\n${artifactIds.join('\n  ')}`);
       }
+
       return [];
     } catch (err) {
+      this.logger.debug(
```

Comment:
should this be an error? Not sure but want to double check

Reply:
good catch. yes, this should be `.error()`. I'll correct it.